kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/SectionSidePanel.vue

```diff
@@ -5,10 +5,18 @@
     ref="resourcePanel"
     alignment="right"
     sidePanelWidth="700px"
-    :closeButtonIconType="closeBackIcon"
+    closeButtonIconType="close"
```

Comment:
We're injecting the "back" icon where/when we want it now into the `#header` slot, so we hard-code "close" here so that the X is always there.